auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.kt

```diff
@@ -96,12 +96,35 @@ public class CredentialsManager @VisibleForTesting(otherwise = VisibleForTesting
         scope: String?,
         minTtl: Int,
         parameters: Map<String, String>
+    ): Credentials {
+        return awaitCredentials(scope, minTtl, parameters, false)
+    }
+
+    /**
+     * Retrieves the credentials from the storage and refresh them if they have already expired.
+     * It will throw [CredentialsManagerException] if the saved access_token or id_token is null,
+     * or if the tokens have already expired and the refresh_token is null.
+     * This is a Coroutine that is exposed only for Kotlin.
+     *
+     * @param scope    the scope to request for the access token. If null is passed, the previous scope will be kept.
+     * @param minTtl   the minimum time in seconds that the access token should last before expiration.
+     * @param parameters additional parameters to send in the request to refresh expired credentials
+     * @param forceRefresh This will avoid returning the existing credentials and retrieves a new one even if valid credentials exist.
```

Comment:
```suggestion
     * @param forceRefresh this will avoid returning the existing credentials and retrieves a new one even if valid credentials exist.
```